packages/app/src/app/overmind/namespaces/dashboard/actions.ts

```diff
@@ -9,11 +9,19 @@ import {
 import Fuse from 'fuse.js';
 import { OrderBy, sandboxesTypes } from './state';
 
+const VIEW_MODE_DASHBOARD = 'VIEW_MODE_CODESANDBOX_DASHBOARD';
```

Comment:
this is scoped to the domain, so we can skip the company name :)

Reply:
Done :)